twoliter/embedded/build.Dockerfile

```
@@ -110,10 +115,15 @@ COPY ./packages/${PACKAGE}/ .
 
 COPY --chown=builder --from=rpm-macros-and-bconds /home/builder/generated.* .
 
-# Merge generated bconds with the package spec, and put sources in the right place.
+# Merge generated bconds with the package spec, and put sources in the right place. Set Epoch values
+# for the package and all of its subpackages. Additionally, for packages declaring explicit package versions
+# via "{Requires,Conflicts,Obsoletes}: = ...", ensure the epoch for the specified package is set.
 RUN \
    cat "/usr/lib/rpm/platform/${ARCH}-bottlerocket/macros" generated.rpmmacros > .rpmmacros \
+   && echo -e -n "Epoch: ${BUILD_EPOCH}\n" >> rpmbuild/SPECS/${PACKAGE}.spec \
    && cat generated.bconds ${PACKAGE}.spec >> rpmbuild/SPECS/${PACKAGE}.spec \
+   && sed -i "/%package.*$/a Epoch: ${BUILD_EPOCH}" rpmbuild/SPECS/${PACKAGE}.spec \
```

consider deleting existing `Epoch` lines, and being more rigorous about matches:
```suggestion
   && sed -i '/^Epoch:/d' rpmbuild/SPECS/${PACKAGE}.spec \
   && sed -i "/^%package\s.*$/a Epoch: ${BUILD_EPOCH}" rpmbuild/SPECS/${PACKAGE}.spec \
```

---

I think this fits into the case that we could alter the expectations of a packager by modifying their Epoch value. Would it be better to fail the build than to produce something unexpected?

---

In the latest revision of this PR, I opted not to delete existing `Epoch` lines for two reasons:

1. attempting to delete previous Epoch was resulting in invalid RPM, with an error along the lines of `File must begin with "/": Epoch: `. I didn't dive further becaue I concluded that instead of making this change, we can:
2. By always attempting to set our `Epoch`, we will induce RPM failures for the case where a packager attempts to specify their own `Epoch`. This seems more reasonable than silently changing the value they intend to set as `Epoch`